fast64_internal/f3d/f3d_gbi.py

```diff
@@ -1426,10 +1426,49 @@ def __init__(self, texCSeparate, savePNG, includeDir, exportPath = ''):
 		self.includeDir = includeDir
 		self.exportPath = exportPath
 
+#  SetTileSize Scroll Data
+class FSetTileSizeScrollField:
+	def __init__(self):
+		self.s = 0
+		self.t = 0
+
+def tile_func(direction: str, speed: int, cmdNum: int):
```

Comment:
type annotations don't hurt, for consistency should we move to have them everywhere? (and require future PRs to have them)

Reply:
i think it'd be helpful to start using them where applicable. there are certain classes that are crazy similar to each other and type annotating would prevent you from having to hunt as much